chutney/packaging/local-api-unsecure/src/main/java/com/chutneytesting/SecurityConfiguration.java

```diff
@@ -13,16 +13,15 @@
 import java.util.Arrays;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.Ordered;
-import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
 import org.springframework.security.web.SecurityFilterChain;
 
 @Configuration
 @EnableWebSecurity
-@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
- Why was this annotation removed ?
- Was it unnecessary, or is filter order now handled differently ?
- has this been tested to ensure there are no execution order issues ?